### PR TITLE
Feature/remove djangojs

### PR DESCRIPTION
**Pull request by jake-kent**
tests don't currently pass because there is no JS integration/compilation system reimplemented

### CODE REVIEW

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/base.py [hunk 1]
```diff
@@ -101,14 +101,13 @@ def get_env_setting(setting, default=None):
     'django_extensions',
     'model_utils',
     'pipeline',
-    'djangojs',
     'casper',
```
Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/base.py]: I think we can remove `casper` as well.

**Comment by jschneier** [comment]
Remove from requirements files as well.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-
+default_app_config = 'home.apps.MainAppConfig'
```
Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py]: I'm fairly certain we need to do this as `{{ cookiecutter.repo_name }}.$app_name.apps.$config_name`

Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py]: Also let's not call it `Main` or anything like that.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/apps.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+class AccountConfig(AppConfig):
+	name = "account"
```
Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/apps.py]: Similarly I think we have to name everything `{{ cookiecutter.repo_name }}.$app_name`

Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/apps.py]: Try running it and seeing what happens if you don't do that.

Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/apps.py]: I would _greatly_ enjoy being proved wrong but I don't think I will be.

Comment (jake-kent) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/apps.py]: Was able to build (./test.sh) and run the server within the created project without an issue

**Pull request closed without merging**